src/CronConstants.ts

```diff
@@ -0,0 +1,121 @@
+import {IFieldConstraints} from './types';
```

Comment:
```suggestion
import { IFieldConstraints } from './types';
```

This is a more general recommendation. I personally like when there is extra spacing inside of brackets. Improves a little bit readability.

Reply:
add object-curly-spacing as eslint rule